short-survey-checklist.md

```diff
@@ -0,0 +1,21 @@
+# Checklist for MDN short surveys
+
+This document lists the standard operating procedure for creating MDN short surveys. 
+
+# Mandatory fields
+- [ ] Topic: The topic of the survey
+- [ ] Category: E.g. CSS, JavaScript, Accessibility, documentation
```

Comment:
is it the category of the pages on which the survey will be shown? if so, that should be clarified